src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1284,18 +1284,25 @@ protected override void Validate(object arguments, EngineIntrinsics engineIntrin
                     null, Metadata.ValidateCountNotInArray);
             }
 
+            if (MinLength == MaxLength && len != MaxLength)
```

Comment:
We should to keep the two original fully qualified error ids to avoid a breaking change. But the new one I could to remove:

What is your conclusion about last @mklement0 [comment](https://github.com/PowerShell/PowerShell/pull/3596#issuecomment-296338493)?
If you reject it I shall only remove the ` if (MinLength == MaxLength && len != MaxLength)`.
If you accept please give me right template.